sorc/ncep_post.fd/CLDRAD.f

```diff
@@ -108,8 +108,8 @@ SUBROUTINE CLDRAD
                             FLD_INFO, AVRAIN, THEAT, IFHR, IFMIN, AVCNVC,     &
                             TCLOD, ARDSW, TRDSW, ARDLW, NBIN_DU, TRDLW, IM,   &
                             NBIN_SS, NBIN_OC,NBIN_BC,NBIN_SU,NBIN_NO3,DTQ2,   &
-                            JM, LM, gocart_on, nasa_on, me, rdaod,ISTA, IEND, &
-                            aqf_on
+                            JM, LM, gocart_on, gccpp_on, nasa_on, me, rdaod,  &
+                            ISTA, IEND,aqf_on
```

Comment:
@zhanglikate Please add a change log at the beginning section of this subroutine.